hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/scm/ReconContainerManager.java

```diff
@@ -64,6 +69,22 @@ protected File getContainerDBPath(Configuration conf) {
     return new File(metaDir, RECON_SCM_CONTAINER_DB);
   }
 
+  public void checkAndAddNewContainer(ContainerID containerID,
+                                      DatanodeDetails datanodeDetails)
+      throws IOException {
+    if (!exists(containerID)) {
+      LOG.info("New container {} got from {}.", containerID, datanodeDetails);
```

Comment:
datanodeDetails toString is very verbose. You may choose to just log the hostname/ip here.